src/init-ceph.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
I understand that your intention is to modify the script for both Linux and FreeBSD. If so why removing this line ?